app/resources/v1/streets.js

```diff
@@ -163,13 +163,9 @@ exports.post = async function (req, res) {
     }
     street.creatorId = user ? user.id : ''
 
-    saveStreet()
-      .then(handleCreatedStreet)
-      .catch(handleErrors)
+    saveStreet().then(handleCreatedStreet).catch(handleErrors)
```

Comment:
this change might be due to some linting differences? I do use prettier locally, not sure whether the diff is because my linting isn't conforming or not :thinking: